src/assets/stylesheets/index.scss

```diff
@@ -405,26 +406,36 @@ div {
 
 a {
   color: var(--bs-primary);
-  text-decoration: none;
+  text-decoration: underline;
```

Comment:
I added "Text with a link" widgets on several places in https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://agessler-css-fixes.scrivito-portal-app.pages.dev/colors?_scrivito_workspace_id=q6a733838df01858&_scrivito_display_mode=view . Some combinations still don't show an underline (e.g. a card within a card). Please investigate.

Reply:
Its not easy. We are using card nearly everywhere. In some cases we don't want underline in person cards in "real" content it could be nice. Not easy to see codewise when what to do.

Reply:
I think the fix to the issues is to change the following

https://github.com/Scrivito/scrivito-portal-app/blob/16e3f867573bffc88c721399e8a6d2863c6232fa/src/assets/stylesheets/index.scss#L428

to `.card > a {`

So with that, the "underline of a card, that is a link" is only limited to the top link and not all inner links as well.

Reply:
Please try this out and update the PR if it works.

Reply:
Will be addressed in a follow up PR.